tests/db/model/generatortemplate/create.js

```diff
@@ -205,9 +205,8 @@ describe('tests/db/model/generatortemplate/create.js >', () => {
     _gt.author.email = 'notanemail';
     _gt.author.url = 'invalid url';
     GeneratorTemplate.create(_gt)
-    .then((x) => {
-      console.log(x);
-      done(' Error: Expecting validation error');
+    .then(() => {
```

Comment:
`.then(() => done('Error: Expecting validation error'))`

Reply:
in every test its like only i have not modified this . @iamigo So should I change or not ?